description-templet.js

```diff
@@ -0,0 +1,113 @@
+module.exports = {
```

Comment:
Could you please remove this description-templet file and put all the descriptions in the endpoints.json file? It’s very confusing at the moment as we have 2 duplicate files and it’s hard to follow what’s going on.